eng/versioning/version_client.txt

```diff
@@ -356,7 +356,7 @@ com.azure.resourcemanager:azure-resourcemanager-logz;1.0.0-beta.2;1.0.0-beta.3
 com.azure.resourcemanager:azure-resourcemanager-storagepool;1.0.0-beta.2;1.0.0-beta.3
 com.azure.resourcemanager:azure-resourcemanager-dataprotection;1.0.0-beta.2;1.0.0-beta.3
 com.azure.resourcemanager:azure-resourcemanager-desktopvirtualization;1.0.0-beta.2;1.0.0-beta.3
-com.azure.resourcemanager:azure-resourcemanager-loadtesting;1.0.0-beta.1;1.0.0-beta.2
+com.azure.resourcemanager:azure-resourcemanager-loadtesting;1.0.0-beta.1;1.0.0
```

Comment:
suppose to be: `com.azure.resourcemanager:azure-resourcemanager-loadtesting;1.0.0;1.1.0-beta.1`？